### DISCUSSION TITLE
YimMenu on Linux?

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#1102] [Q&A] [answered]
Hey so recently i have been thinking about migrating to Linux and would there be any way to inject YimMenu into GTA 5 which is running under Proton?

**Comment by Yimura** [comment, marked as answer]
Please see the following issue for more information on how to run YimMenu on Linux #334

**Reply by ghost** [reply]
Thanks! I will try it out after I get Linux up and running :D

**Reply by xM4ddy** [reply]
> Thanks! I will try it out after I get Linux up and running :D

Yeah I can confirm got it running on Linux
